content/en/blog/2023/contributing-to-otel-demo/index.md

```diff
@@ -0,0 +1,1026 @@
+	✔ Payment: invalid credit card (http://tracetest-server:11633/test/payment-invalid-credit-card/run/2/test) - trace id: a20c1d166cd6edf4d8f288e407e76623
+		✔ It should call Charge method and receive a gRPC error
+		✔ It should return a return an gRPC error code to the caller
+	✔ Payment: Amex credit card not allowed (http://tracetest-server:11633/test/payment-amex-credit-card-not-allowed/run/2/test) - trace id: a5812bf50ac61a387dc991ba0dd3020a
+		✔ It should call Charge method and receive a gRPC error
+		✔ It should return a return an gRPC error code to the caller
+	✔ Payment: expired credit card (http://tracetest-server:11633/test/payment-expired-credit-card/run/2/test) - trace id: fecb6368104b0fc27e7806136fc1ab1c
+		✔ It should call Charge method and receive a gRPC error
+		✔ It should return a return an gRPC error code to the caller
+
+Tests done! Exit code: 0
+```
+
+You can also start the Tracetest services alongside all the Core Demo Services,
+Dependent Services, and Telemetry Components as part of your development
+lifecycle to enable
+[Observability-driven Development (ODD)](https://stackoverflow.blog/2022/10/12/how-observability-driven-development-creates-elite-performers/).
+Mainly to trigger your APIs and validate both the response and trace data they
+generate. You can also build tests visually and save them as YAML files to add
+to your code repository for automated testing. I’ll walk you through all of this
+a bit later as well.
+
+To do this, you’ll use the `odd`
+[Docker Compose profile](https://docs.docker.com/compose/profiles/). Run the
+demo like this:
+
+```bash
+docker compose --profile odd up --force-recreate --remove-orphans --detach
+
+# OR
+
+make start-odd
+```
+
+Go ahead and start the OpenTelemetry Demo including Tracetest, with the `odd`
+profile.
+
+Once the images are built and containers are started you can access:
+
+- Web store: [http://localhost:8080/](http://localhost:8080/)
+- Grafana: [http://localhost:8080/grafana/](http://localhost:8080/grafana/)
+- Feature Flags
+  UI: [http://localhost:8080/feature/](http://localhost:8080/feature/)
+- Load Generator
+  UI: [http://localhost:8080/loadgen/](http://localhost:8080/loadgen/)
+- Jaeger
+  UI: [http://localhost:8080/jaeger/ui/](http://localhost:8080/jaeger/ui/)
+- Tracetest UI: [http://localhost:11633/](http://localhost:11633/), only when
+  using `make start-odd`
+
+To run a test against the Payment Service, I’ll use a YAML file and trigger it
+with the
+[Tracetest CLI](https://docs.tracetest.io/getting-started/installation#install-the-tracetest-cli).
+Alternatively, you can also build tests visually in the Tracetest UI
+on [http://localhost:11633/](http://localhost:11633/).
+
+> _Here’s a
+> [short guide on how to create tests programatically](https://docs.tracetest.io/getting-started/open#creating-trace-based-tests-programatically)
+> with Tracetest._
+
+Next, let’s move on to explaining and adding code instrumentation in the
+`paymentservice`.
```

Comment:
Can all of this be replaced with a reference to https://opentelemetry.io/docs/demo/ ? It duplicates a lot of words from there